etna/lib/etna/route.rb

```diff
@@ -148,7 +148,10 @@ def user_authorized?(request)
       params = request.env['rack.request.params']
 
       @auth[:user].all? do |constraint, param_name|
-        user.respond_to?(constraint) && user.send(constraint, params[param_name])
+        user.respond_to?(constraint) && (
+          params.has_key?(param_name) ?
+            user.send(constraint, params[param_name]) :
+            user.send(constraint, param_name))
```

Comment:
Maybe safer to do this on `param_name.is_a?(Symbol)` rather than params.has_key?, since that is subject to the user input.